grafana/grafana.spec

```diff
@@ -0,0 +1,309 @@
+Provides: bundled(nodejs-minimist) = 1.2.0
+Provides: bundled(nodejs-nopt) = 3.0.6
+Provides: bundled(nodejs-normalize-package-data) = 2.4.2
+Provides: bundled(nodejs-number-is-nan) = 1.0.1
+Provides: bundled(nodejs-object-assign) = 4.1.1
+Provides: bundled(nodejs-once) = 1.4.0
+Provides: bundled(nodejs-parse-json) = 2.2.0
+Provides: bundled(nodejs-path-exists) = 2.1.0
+Provides: bundled(nodejs-path-is-absolute) = 1.0.1
+Provides: bundled(nodejs-path-type) = 1.1.0
+Provides: bundled(nodejs-pify) = 2.3.0
+Provides: bundled(nodejs-pinkie) = 2.0.4
+Provides: bundled(nodejs-pinkie-promise) = 2.0.1
+Provides: bundled(nodejs-read-pkg) = 1.1.0
+Provides: bundled(nodejs-read-pkg-up) = 1.0.1
+Provides: bundled(nodejs-redent) = 1.0.0
+Provides: bundled(nodejs-repeating) = 2.0.1
+Provides: bundled(nodejs-resolve) = 1.1.7
+Provides: bundled(nodejs-rimraf) = 2.2.8
+Provides: bundled(nodejs-safer-buffer) = 2.1.2
+Provides: bundled(nodejs-semver) = 5.6.0
+Provides: bundled(nodejs-signal-exit) = 3.0.2
+Provides: bundled(nodejs-spdx-correct) = 3.1.0
+Provides: bundled(nodejs-spdx-exceptions) = 2.2.0
+Provides: bundled(nodejs-spdx-expression-parse) = 3.0.0
+Provides: bundled(nodejs-spdx-license-ids) = 3.0.3
+Provides: bundled(nodejs-sprintf-js) = 1.0.3
+Provides: bundled(nodejs-strip-ansi) = 3.0.1
+Provides: bundled(nodejs-strip-bom) = 2.0.0
+Provides: bundled(nodejs-strip-indent) = 1.0.1
+Provides: bundled(nodejs-supports-color) = 2.0.0
+Provides: bundled(nodejs-trim-newlines) = 1.0.0
+Provides: bundled(nodejs-underscore.string) = 3.2.3
+Provides: bundled(nodejs-validate-npm-package-license) = 3.0.4
+Provides: bundled(nodejs-which) = 1.2.14
+Provides: bundled(nodejs-wrappy) = 1.0.2
+Provides: bundled(nodejs-yarn) = 1.13.0
+
+
+%description
+Grafana is an open source, feature rich metrics dashboard and graph editor for
+Graphite, InfluxDB & OpenTSDB.
+
+%package prometheus
+Requires: %{name} = %{version}-%{release}
+Summary: Grafana prometheus datasource
+
+%description prometheus
+The Grafana prometheus datasource.
+
+%prep
+%setup -q -T -D -b 0
+%setup -q -T -D -b 1
+%patch1 -p1
+%patch2 -p1
+%patch3 -p1
+
+# Set up build subdirs and links
+mkdir -p %{_builddir}/src/github.com/grafana
+ln -sf %{_builddir}/%{name}-%{version} \
+    %{_builddir}/src/github.com/grafana/grafana
+
+# remove some (apparent) development files, for rpmlint
+rm -f public/sass/.sass-lint.yml public/test/.jshintrc
+
+%build
+# Build the server-side binaries: grafana-server and grafana-cli
+%if 0%{?gobuild}
+# use modern go macros such as in recent Fedora
+export GOPATH=%{_builddir}:%{gopath}
+%gobuild -o grafana-cli ./pkg/cmd/grafana-cli
+%gobuild -o grafana-server ./pkg/cmd/grafana-server
+%else
+cd %{_builddir}/src/github.com/grafana/grafana
+export GOPATH=%{_builddir}:%{gopath}
+go run build.go build
+%endif
+
+
+%install
+# Fix up arch bin directories
+[ ! -d bin/x86_64 ] && ln -sf linux-amd64 bin/x86_64
+[ ! -d bin/i386 ] && ln -sf linux-386 bin/i386
```

Comment:
You will likely need the binary stripping logic. not seeing it

Reply:
Added here: https://github.com/Maistra/rpms/pull/43/files#diff-5d5c344596cac436a5cbba3c5b4426b0R193